app/src/main/res/layout/fragment_course_detail.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You have a lot of nested layouts which could be optimized using ConstraintLayout:
https://codelabs.developers.google.com/codelabs/constraint-layout/index.html#0